products/PurePhone/sys/SystemManager.cpp

```diff
@@ -32,11 +32,12 @@ namespace sys
 
     void SystemManager::StartSystem(InitFunction sysInit, InitFunction appSpaceInit, DeinitFunction sysDeinit)
     {
-        auto simSelected = []() {
-            return (Store::GSM::get()->selected == Store::GSM::SIM::SIM1 ||
-                    Store::GSM::get()->selected == Store::GSM::SIM::SIM2);
+        auto activeSimSelected = []() {
+            const auto storeGsm = Store::GSM::get();
+            return ((storeGsm->selected == Store::GSM::SIM::SIM1 || storeGsm->selected == Store::GSM::SIM::SIM2) &&
+                    storeGsm->simCardInserted());
```

Comment:
this is the fix, the rest is rename noise :)

Reply:
3 changed files it not a much noise - one day there were 15+ files with renames only ;)